apps/templates/content_templates.py

```diff
@@ -506,7 +506,12 @@ def render_content_template(item, template, update=False):
     :param dict template: template
     :return dict: updates to the item
     """
+
     kwargs = dict(item=item, user=get_user())
+    dateline_present_in_user_preferences = (kwargs['user'].get('user_preferences', {})
+                                            .get('dateline:located', {}).get('located'))
+    if dateline_present_in_user_preferences:
+        TEMPLATE_DATA_IGNORE_FIELDS.add('dateline')
```

Comment:
once you add it there it will use that for all following requests no matter if dateline will be in user preferences or not. imo it should make a copy of the fields and then add it just for this call